src/main/kotlin/no/nav/familie/tilbake/domain/FaktaFeilutbetalingsperiode.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.familie.tilbake.domain
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Embedded
+import java.time.LocalDate
+import java.util.*
+
+data class FaktaFeilutbetalingsperiode(@Id
+                                       val id: UUID = UUID.randomUUID(),
+                                       val faktaFeilutbetalingId: UUID?,
+                                       val fom: LocalDate,
+                                       val tom: LocalDate,
+                                       val hendelsestype: Hendelsestype,
+                                       val hendelsesundertype: Hendelsesundertype,
+                                       @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
+                                       val sporbar: Sporbar = Sporbar())
+
+enum class Hendelsestype(val navn: String, val sortering: Int) {
+    MEDLEMSKAP_TYPE("§14-2 Medlemskap", 10),
+    ØKONOMI_FEIL("Feil i økonomi", 500),
+    PSB_ANNET_TYPE("Annet", 999),
+    PPN_ANNET_TYPE("Annet", 999),
+    OLP_ANNET_TYPE("Annet", 999)
+}
+
+enum class Hendelsesundertype(val navn: String?, val sortering: Int) {
+    IKKE_INNTEKT("Ikke inntekt 6 av siste 10 måneder", 0),
+    IKKE_YRKESAKTIV("Ikke yrkesaktiv med pensjonsgivende inntekt", 1),
+    INNTEKT_UNDER("Inntekt under 1/2 G", 1),
+    ENDRING_GRUNNLAG("Endring i selve grunnlaget", 0),
+    ENDRET_DEKNINGSGRAD("Endret dekningsgrad", 0),
+}
```

Comment:
Her må vi nok tenke oss litt om.
1. Forskjellige ytelser skal ha hver sin liste med Hendelsestyper, så kanskje legge til "val ytelseType: ytelseType) også ha en egen ENUM et sted for de forskjellige ytelseType (BA, EF og KS)
I tillegg er dataen er gammel, PP er vel Pleiepenger og OL er vel opplæringspenger. Pr dags dato har vi ingen liste over hendelser for familie-ytelsene, så listen for Hendelsestype burde bare vært BA_ANNET_TYPE, EF_ANNET_TYPE og KS_ANNET_TYPE mens medlem og øko kan fjernes.
2. Hendelsesundertype skal knyttes til en Hendelsestype. "val hendelseType: Hendelsetype" ?

Reply:
Jeg er litt skeptisk til å legge til ytelse på Hendelsestype. Da risikerer vi vel å ende opp med å duplisere hensdelsestyper, fordi det viser seg at økonomifeil er aktuelt både for BA og EF f.eks.
Samme tenker jeg om hendelseundertype.